.github/workflows/ciab.yaml

```diff
@@ -0,0 +1,194 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: CDN-in-a-Box CI
+
+on:
+  push:
+  create:
+  pull_request:
+    types: [opened, reopened, edited, synchronize]
+
+jobs:
+  source:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@master
+    - name: Build RPM
+      uses: ./.github/actions/build-rpms
+      env:
+        ATC_COMPONENT: ${{ github.job }}
+    - name: Upload RPM
+      uses: actions/upload-artifact@v2
+      with:
+        name: ${{ github.job }}
+        path: ${{ github.workspace }}/dist/apache-trafficcontrol-*.tar.gz
+
+  traffic_monitor:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@master
+    - name: Build RPM
+      uses: ./.github/actions/build-rpms
+      env:
+        ATC_COMPONENT: ${{ github.job }}
+    - name: Upload RPM
+      uses: actions/upload-artifact@v2
+      with:
+        name: ${{ github.job }}
+        path: ${{ github.workspace }}/dist/${{ github.job }}-*.rpm
```

Comment:
if you aren't worried about hosting source RPMs as artifacts, it might be easier to archive `infrastructure/cdn-in-a-box/${{ github.job }}/${{ github.job }}.rpm`

Reply:
That's an option, but then we'd need to do something different in the `build-rpms` action for `grove` and `grovetccfg`, since neither of those components are used in CDN-in-a-Box.

Reply:
oh, fair enough. I guess just be aware that this will create two artifacts for each component - one being much (?) larger - or at least that's what it looks like without testing it.

Reply:
Why do we make source RPMs anyway? We're doing `rpmbuild -ba`, but we could just as easily do `rpmbuild -bb` and build binary RPMs only.